src/duration/extractMinMaxInUnits.js

```diff
@@ -1,5 +1,5 @@
 import durationUnits from './durationUnits';
-import splitSecsToUnits from './splitSecsToUnits';
+import splitSecsToUnits from '../date/splitSecsToUnits';
```

Comment:
I intentionally did it so we don't have everything in index.js and export because then can not be optimized out, if you include one thing from index.js like import { something } from binary-utils then all funcs are compiled in.

Might still do it though and skip the /lib/ thing.
What do you think?